### DISCUSSION TITLE
character jittering after armature to riggfy retargeting

### DISCUSSION THREAD

**Discussion by MOTO4GIT** [soupday/cc3_blender_tools#130] [General]
the character's full body will jitter after the retargeting and the jaw bone will jitter very drastically.

**Comment by buttabean** [comment]
Did you find a solution to this problem?

**Comment by buttabean** [comment]
I was able to pinpoint the issue. Any coordinate set to the character in iclone that's off 0 causes an issue. It seems it's not noticeable until you go to 150+- xyz . it seems linear so the further away from 0, the worse it becomes.  It's an unfortunate bug that seems to be around a while.

**Reply by MOTO4GIT** [reply]
Yes, farther from the world origin, more jitter to the rig, not just the jaw, every point is jittering, so for bigger scenes, the ultimate way is to stay in the FK armature, or you have to smooth all control rigs that jittering obviously
